<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScores</class>
 <widget class="QWidget" name="HighScores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="highScoresTableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>High Scores</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearHighScoresBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear High Scores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highScoresPerMatchBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>High Scores Per Match</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highTotalScorePerUserBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>High Total Score Per User</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>highScoresTableView</zorder>
  <zorder>clearHighScoresBtn</zorder>
  <zorder>highScoresPerMatchBtn</zorder>
  <zorder>highTotalScorePerUserBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
